pkg/client/observatorium/observability_config.go

```diff
@@ -22,31 +21,19 @@ const (
 )
 
 type ObservabilityConfiguration struct {
-	// Dex configuration
-	DexUrl          string `json:"dex_url" yaml:"dex_url"`
-	DexUsername     string `json:"username" yaml:"username"`
-	DexPassword     string `json:"password" yaml:"password"`
-	DexSecret       string `json:"secret" yaml:"secret"`
-	DexSecretFile   string `json:"secret_file" yaml:"secret_file"`
-	DexPasswordFile string `json:"password_file" yaml:"password_file"`
-
 	// Red Hat SSO configuration
 	RedHatSsoAuthServerUrl     string `json:"redhat_sso_auth_server_url" yaml:"redhat_sso_auth_server_url"`
 	RedHatSsoRealm             string `json:"redhat_sso_realm" yaml:"redhat_sso_realm"`
 	RedHatSsoTenant            string `json:"redhat_sso_tenant" yaml:"redhat_sso_tenant"`
 	RedHatSsoTokenRefresherUrl string `json:"redhat_sso_token_refresher_url" yaml:"redhat_sso_token_refresher_url"`
 
 	// Observatorium configuration
-	ObservatoriumGateway string        `json:"gateway" yaml:"gateway"`
-	ObservatoriumTenant  string        `json:"observatorium_tenant" yaml:"observatorium_tenant"`
-	AuthType             string        `json:"auth_type" yaml:"auth_type"`
-	AuthToken            string        `json:"auth_token"`
-	AuthTokenFile        string        `json:"auth_token_file"`
-	Cookie               string        `json:"cookie"`
-	Timeout              time.Duration `json:"timeout"`
-	Insecure             bool          `json:"insecure"`
-	Debug                bool          `json:"debug"`
-	EnableMock           bool          `json:"enable_mock"`
+	AuthType   string        `json:"auth_type" yaml:"auth_type"`
+	Cookie     string        `json:"cookie"`
```

Comment:
Do we still need this? It looks like it was only an option when the auth type was Dex.

Reply:
You are right :+1: . It is still referenced in the Observatorium client code but then it is not used anywhere. I will remove it.

Reply:
Updated